logics/impls/controller/utils.rs

```diff
@@ -140,6 +140,10 @@ pub fn get_hypothetical_account_liquidity(
         borrow_amount,
     } = input;
 
+    if asset_params.is_empty() {
+        sum_borrow_plus_effect = U256::from(redeem_tokens + borrow_amount);
+    }
```

Comment:
Why is this necessary?
Is this different from what is considered in the following codes that follow?
```rust
if token_modify.is_some() && param.asset == token_modify.unwrap() { ... }
```

Reply:
If asset_params is empty vector, sum_borrow is gettin 0 and not correct.

Reply:
I understand what you are concerned about.
It just doesn't seem to be scaled or converted to the subsequent scale.

```rust
            // redeem effect
            // sumBorrowPlusEffects += tokensToDenom * redeemTokens
            sum_borrow_plus_effect = token_to_denom.clone().mul_scalar_truncate_add_uint(
                to_flatten(U256::from(redeem_tokens)),
                sum_borrow_plus_effect,
            );

            // borrow effect
            // sumBorrowPlusEffects += oraclePrice * borrowAmount
            sum_borrow_plus_effect = param
                .oracle_price_mantissa
                .clone()
                .mul_scalar_truncate_add_uint(
                    to_flatten(U256::from(borrow_amount)),
                    sum_borrow_plus_effect,
                );
```

Also, it seems that this function is currently only used in logics/impls/controller/mod#_get_hypothetical_account_liquidity
If there is a pattern where asset_params is empty but token_modify has a value, it needs to be fixed immediately.